shardingsphere-infra/shardingsphere-infra-common/src/main/java/org/apache/shardingsphere/infra/storage/StorageNodeStatus.java

```diff
@@ -15,12 +15,12 @@
  * limitations under the License.
  */
 
-package org.apache.shardingsphere.mode.manager.cluster.coordinator.registry.status.storage;
+package org.apache.shardingsphere.infra.storage;
```

Comment:
why move to infra？

Reply:
Because, I think storage node status needs to be passed in when use `readwrite-splitting` or `db-discovery`. Move `StorageNodeStatus` to infra module. It is more convenient to used.